### DISCUSSION TITLE
如何管理自己的时间资产 · /few-words-time-management

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#597] [Announcements]
https://draveness.me/few-words-time-management 

 时间是我们最重要的资产，对时间的使用和管理决定了我们的个人成长。人的注意力是有限的，我们不能将同一份时间投入到不同的事情中，一旦选择花费时间做一些事情，就不得不放弃做其他事情的可能性。如果我们能够静下心来思考应该如何使用自己的最重要资产，一定能获得更大的个人成长和回报。时间管理是一个非常大的话题，每个人都使用不同的工具和思想来管理和分配时间，因为之前有读者希望作者分享一下时间管理的经验，所以在这里我也跟各位读者简单分享一下自己在时间管理上的方法。

**Comment by Friday21** [comment]
受益匪浅， 看到大佬每月都能产出这么高质量的文章，真心感到佩服。

**Comment by superdoliant** [comment]
大佬在哪个公司呀？现在找非996的公司好难呀

**Comment by mrsandwitch** [comment]
想请问第七点：执行 Sprint 的过程中发现任务预估时间与期望值过大应该及时调整计划，保证高优任务

详细操作怎么操作，是发现任务比预期花时间先改做其他任务的吗？还是把任务拆分到下个 sprint ？还是把其他计划排开？

**Comment by draveness** [comment]
> 大佬在哪个公司呀？现在找非996的公司好难呀

主要还是要看老板，大公司不同部门情况也非常不一样

**Comment by draveness** [comment]
> 想请问第七点：执行 Sprint 的过程中发现任务预估时间与期望值过大应该及时调整计划，保证高优任务
> 
> 详细操作怎么操作，是发现任务比预期花时间先改做其他任务的吗？还是把任务拆分到下个 sprint ？还是把其他计划排开？

直接把低优先级的任务从这一次计划中删除

**Comment by Rwantop** [comment]
学习了！  作者真是高产文章又高质量，受益匪浅。

**Comment by wmyskxz** [comment]
不才，小小推荐一下 Notion 这款软件

**Comment by hurryking** [comment]
为什么系列真的很棒，感谢大佬

**Comment by XinruZhang** [comment]
我是看《软技能 代码之外的生存指南》学到的用 Trello，真心有效，颜值又高

**Comment by draveness** [comment]
> 我是看《软技能 代码之外的生存指南》学到的用 Trello，真心有效，颜值又高

Trello 打钱

**Comment by BryceLee** [comment]
请问Trello 如何让label显示出文字？

**Comment by draveness** [comment]
> 请问Trello 如何让label显示出文字？

点一下那个 Label

**Comment by lexiaoyao20** [comment]
请问大佬你那个 Why‘s the Design 和 blog 标签文字是怎么显示出来的，我这边设置一个标签只有一个颜色条

**Comment by draveness** [comment]
> 请问大佬你那个 Why‘s the Design 和 blog 标签文字是怎么显示出来的，我这边设置一个标签只有一个颜色条

看上面

**Comment by swordphp** [comment]
带劲,感觉发现各种宝藏.

**Comment by jinghzhu** [comment]
我也是和博主一样的时间管理方式好几年，甚至还引入精力管理方式。

看了博主的高产和高质量后，结合自己的经验，想告诉大家的是。如果你也像博主这样尽管在B厂这样的996公司还能保住自己快速学习，那么必须大幅压缩自己各种课余时间（陪女友、处理家庭事务、自己的社交和兴趣等等）。夸张点说，你的生活多数情况只有三种活动：上班、学习和睡觉。

原因很简单。所有人每天都只有24小时，所有人不可能每个小时都处在精力旺盛效率较高的状态，再怎么努力学习管理自己的时间和精力，但终究要被客观物理条件限制。能做的，只能是拆东墙补西墙，即压缩其它活动时间。

所以，如果你觉得你能这样坚持，这也是你想要的生活，那么可以按照这种方式精细化管理自己的目标和时间。否则，还是定个high level的学习OKR追踪更适合。

**Comment by draveness** [comment]
> 看了博主的高产和高质量后，结合自己的经验，想告诉大家的是。如果你也像博主这样尽管在B厂这样的996公司还能保住自己快速学习，那么必须大幅压缩自己各种课余时间（陪女友、处理家庭事务、自己的社交和兴趣等等）。夸张点说，你的生活多数情况只有三种活动：上班、学习和睡觉。

找个不 996 的公司或者组呗

**Comment by Huweicai** [comment]
> @draveness
> > 大佬在哪个公司呀？现在找非996的公司好难呀
> 
> 主要还是要看老板，大公司不同部门情况也非常不一样

同意大佬的说法

**Comment by wongtseng** [comment]
大致看了几篇博客，质量都很高，关注并佩服！，文章先收藏，需要花点时间研究。

**Comment by minicloudsky** [comment]
大佬方便问下你博客是基于github issues实现的吗，感觉主题很不错，方便的话可以用下你这个主题吗，谢谢了~

**Comment by 7u** [comment]
请问用的是trello付费版吗？

**Comment by haiker2011** [comment]
大佬经常使用的绘图软件是什么呀？

**Comment by guowei-gong** [comment]
：）早上 8 点起床，9 点半开始工作，在地铁上有 30 分钟，然后晚上 18 点下班，每个工作日都会有 1 小时的地铁时间以及晚上 19:30 ~ 22:00 的 2.5 个小时，非工作日每天可以分配 6 小时，那么每周就有 29.5 小时的时间支配。一年有 4 个季度，一个季度有 3 个月，每个月有 4 个星期。两个星期作为一个 Sprint 迭代周期，这两周我有 59 小时的时间完成任务。

**Comment by draveness** [comment]
> ：）早上 8 点起床，9 点半开始工作，在地铁上有 30 分钟，然后晚上 18 点下班，每个工作日都会有 1 小时的地铁时间以及晚上 19:30 ~ 22:00 的 2.5 个小时，非工作日每天可以分配 6 小时，那么每周就有 29.5 小时的时间支配。一年有 4 个季度，一个季度有 3 个月，每个月有 4 个星期。两个星期作为一个 Sprint 迭代周期，这两周我有 59 小时的时间完成任务。

只能说牛逼，但是建议劳逸结合
